railties/lib/rails/generators/rails/app/templates/config/databases/sqlite3.yml.tt

```diff
@@ -20,6 +20,13 @@ test:
   <<: *default
   database: storage/test.sqlite3
 
+
+# SQLite3 write its data on the local filesystem, as such it requires
+# persistent disks. If you are deploying to a managed service, you should
+# make sure it provides disk persistence, as many don't.
+#
+# Similarly, if you deploy your application as a Docker container, you must
+# ensure the database is located in a persisted volume.
 production:
   <<: *default
-  database: storage/production.sqlite3
+# database: path/to/persistent/storage/production.sqlite3
```

Comment:
```suggestion
# database: storage/production.sqlite3
```
I think just commenting it out is fine since most would probably use the default storage folder anyway, but persisted with something like [Docker volumes](https://kamal-deploy.org/docs/configuration#:~:text=path%3A/container/path%22-,Using%20directories).